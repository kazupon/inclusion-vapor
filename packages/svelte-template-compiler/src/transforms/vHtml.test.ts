import { expect, test } from 'vitest'

test.todo('@html', () => {
  const source = `<div>{@html code}</div>`
  expect(source).toBe('todo')
})
