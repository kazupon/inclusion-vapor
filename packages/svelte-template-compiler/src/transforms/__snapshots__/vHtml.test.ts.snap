// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`basic > expected 1`] = `
"import { renderEffect as _renderEffect, setHtml as _setHtml, template as _template } from 'vue/vapor';
const t0 = _template("<div></div>")

export function render(_ctx) {
  const n0 = t0()
  _renderEffect(() => _setHtml(n0, code))
  return n0
}"
`;

exports[`basic > received 1`] = `
"import { renderEffect as _renderEffect, setHtml as _setHtml, template as _template } from 'vue/vapor';
const t0 = _template("<div></div>")

export function render(_ctx) {
  const n1 = t0()
  const n0 = n1.firstChild
  _renderEffect(() => _setHtml(n0, code))
  return n1
}"
`;
